src/chat/ChatLogic.ts

```diff
@@ -225,7 +225,7 @@ export class ChatLogic {
     }
     const inviteBody = `
 <> a <http://www.w3.org/ns/pim/meeting#LongChatInvite> ;
-${this.ns.rdf("seeAlso")} <${chatThing.value}> . 
```

Comment:
**issue** Unfortunately the corresponding test does a character-exact match and broke. Please remove the space in the test expectation as well.

Reply:
Done